docsets/Phaser-CE/docset.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Phaser-CE",
+  "version": "2.20.1",
+  "archive": "Phaser-CE.tgz",
```

Comment:
This file doesn't exist.

Reply:
I thought I had to append the .txt to the actual file (Phaser-CE.tgz.txt), but omit the .txt here?